fence/resources/user/user_session.py

```diff
@@ -59,8 +59,10 @@ def __init__(self, session_token):
                 # if session token is invalid, create a new
                 # empty one silently
                 jwt_info = self._get_initial_session_token()
-        else:
+        elif access_token:
             jwt_info = self._get_initial_session_token()
+        else:
+            jwt_info = None
```

Comment:
```suggestion
            jwt_info = {"context": {}}
```
this is what it sets it to when `clear` is called. things like https://github.com/uc-cdis/fence/blob/master/fence/resources/user/user_session.py#L334 will raise exception if this is `None`

Reply:
oooooh thank you